backend/routes/users.js

```diff
@@ -117,6 +117,15 @@ router.post("/login", async (req, res) => {
           message: `You are logged in ${user.firstname}.`,
           id: user._id,
           accessToken: user.accessToken,
+          // Do we need to add all of the user data back to the frontend?
```

Comment:
No we don't because here we get the accessToken. We will need to get the user info from the get. In the store we will have to make 3 more logics one for get were we do this one for patch and one for delete.